builder/Makefile

```diff
@@ -15,14 +15,18 @@ prelastword = $(word $(words $1),_ $1)
 cname_version = $(call lastword,$(subst -, ,$1))
 cname_arch = $(call prelastword,$(subst -, ,$1))
 
-define require_var =
-ifndef $1
-$$(error '$1 undefined')
+ifeq ($(REPO),)
+        override REPO := http://deb.debian.org/debian
```

Comment:
maybe these defaults would be simpler to set inside the build scripts

Reply:
you're right. I have changed it.